test/SampleConnectorUnitTests/SampleConnectorUnitTests.csproj

```diff
@@ -57,67 +57,65 @@
-      <HintPath>..\..\src\packages\Autodesk.DataExchange.3.3.1-beta\lib\net48\Autodesk.DataExchange.Schemas.dll</HintPath>
+    <Reference Include="Autodesk.DataExchange.Schemas, Version=4.0.4.0, Culture=neutral, processorArchitecture=AMD64">
+      <HintPath>..\..\src\packages\Autodesk.DataExchange.4.0.4-beta\lib\net48\Autodesk.DataExchange.Schemas.dll</HintPath>
     </Reference>
-    <Reference Include="Autodesk.DataExchange.SourceProvider, Version=3.3.1.0, Culture=neutral, processorArchitecture=AMD64">
-      <HintPath>..\..\src\packages\Autodesk.DataExchange.3.3.1-beta\lib\net48\Autodesk.DataExchange.SourceProvider.dll</HintPath>
+    <Reference Include="Autodesk.DataExchange.SourceProvider, Version=4.0.4.0, Culture=neutral, processorArchitecture=AMD64">
+      <HintPath>..\..\src\packages\Autodesk.DataExchange.4.0.4-beta\lib\net48\Autodesk.DataExchange.SourceProvider.dll</HintPath>
     </Reference>
-    <Reference Include="Autodesk.DataExchange.UI, Version=3.3.1.0, Culture=neutral, processorArchitecture=AMD64">
-      <HintPath>..\..\src\packages\Autodesk.DataExchange.UI.3.3.1-beta\lib\net48\Autodesk.DataExchange.UI.dll</HintPath>
+    <Reference Include="Autodesk.DataExchange.UI, Version=4.0.4.0, Culture=neutral, processorArchitecture=AMD64">
+      <HintPath>..\..\src\packages\Autodesk.DataExchange.UI.4.0.4-beta\lib\net48\Autodesk.DataExchange.UI.dll</HintPath>
     </Reference>
-    <Reference Include="Autodesk.DataExchange.UI.ViewModels, Version=3.3.1.0, Culture=neutral, processorArchitecture=AMD64">
-      <HintPath>..\..\src\packages\Autodesk.DataExchange.UI.3.3.1-beta\lib\net48\Autodesk.DataExchange.UI.ViewModels.dll</HintPath>
+    <Reference Include="Autodesk.DataExchange.UI.ViewModels, Version=4.0.4.0, Culture=neutral, processorArchitecture=AMD64">
+      <HintPath>..\..\src\packages\Autodesk.DataExchange.UI.4.0.4-beta\lib\net48\Autodesk.DataExchange.UI.ViewModels.dll</HintPath>
     </Reference>
     <Reference Include="Autodesk.Extensions.Http.ForgeRetry, Version=8.0.0.0, Culture=neutral, processorArchitecture=MSIL">
       <HintPath>..\..\src\packages\Autodesk.Extensions.Http.ForgeRetry.8.0.0\lib\netstandard2.0\Autodesk.Extensions.Http.ForgeRetry.dll</HintPath>
     </Reference>
     <Reference Include="Autodesk.Forge, Version=1.9.8.0, Culture=neutral, processorArchitecture=MSIL">
       <HintPath>..\..\src\packages\Autodesk.Forge.1.9.8\lib\net48\Autodesk.Forge.dll</HintPath>
     </Reference>
-    <Reference Include="Autodesk.GeometryPrimitives, Version=0.5.7.0, Culture=neutral, processorArchitecture=AMD64">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\src\packages\Autodesk.DataExchange.3.3.1-beta\build\native\libs\Autodesk.GeometryPrimitives.dll</HintPath>
+    <Reference Include="Autodesk.GeometryPrimitives">
+      <HintPath>..\..\src\packages\Autodesk.DataExchange.4.0.3-beta\build\native\libs\Autodesk.GeometryPrimitives.dll</HintPath>
```

Comment:
@babshev , can we update the geometry packages here to refer from 4.0.4-beta folder?

Reply:
Thanks @rinku-thakur. I have changed to 4.0.4-beta